lib/ruby/lib/alces/stack/status/run.rb

```diff
@@ -19,138 +19,116 @@
 # For more information on the Alces Metalware, please visit:
 # https://github.com/alces-software/metalware
 #==============================================================================
-require 'alces/tools/execution'
 require 'alces/tools/cli'
 require 'alces/stack/iterator'
-require 'alces/stack/status/monitor'
-require 'alces/stack/status/task'
-require 'alces/stack/iterator'
 require 'alces/stack/log'
+require 'alces/stack/options'
+require 'alces/stack/status/monitor'
+require 'alces/stack/status/job'
 require 'fileutils'
 
 module Alces
   module Stack
+    class RunOptions < Alces::Stack::Options 
+      def cmds
+        [:power, :ping]
+      end
+
+      def nodes
+        @nodes ||= _set_nodes
+      end
+
+      def _set_nodes
+        lambda_proc = lambda { |lambda_hash| lambda_hash[:nodename] }
+        a = Alces::Stack::Iterator.run(group, lambda_proc, nodename: nodename)
+        a.is_a?(Array) ? a : [a]
```

Comment:
I think this is irrelevant/resolved with the new `Nodes` class, don't worry about this